proposals/nnnn-secondary-transport-optimization.md

```diff
@@ -0,0 +1,62 @@
+# Secondary Transport Optimization
+
+* Proposal: [SDL-NNNN](nnnn-secondary-transport-optimization.md)
+* Author: [Joey Grover](https://github.com/joeygrover)
+* Status: **Awaiting Review**
+* Impacted Platforms: [Core / iOS / Android]
+
+## Introduction
+
+This proposal aims to optimize how secondary transports are connected and registered. The original proposal [SDL-0141](0141-multiple-transports.md) stated secondary transports should be connected to and registered on immediately when they are available. This proposal is to change that to connect and register on when necessary by the application.
```

Comment:
Change to: 
This proposal is to change that to connect and register only when necessary by the application.